### DISCUSSION TITLE
Simulation of Spray Fire

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13748] [Q&A]
Hello team,
I wanna do a simulation of spray fire by FDS6.9.1, but based on my understanding, there's only suppression model for exdinction situations. In my case, sodium liquid with high temperature(over 300℃) spray from a pipe, when it exposed to air it fired. Could you please give me some advice on how to simulate this process?
Thanks!
LI

**Comment by mcgratta** [comment]
Why don't you do a simple Internet search to get started. I did, and the first hit was a paper entitled "Liquid spray modeling under sodium fire accidents". Once you educate yourself on the subject, and then learn a bit more about the FDS model, then you may ask more specific and detailed questions here in this forum.

**Reply by lihy213** [reply]
Yes I indeed did this, but the problem was that rarely had research to do with FDS spray fire. Could you please give some guidance so that I could have a quick start?

**Reply by mcgratta** [reply]
You asked about sodium liquid spray fires. I know nothing about this. Let's see if anyone else does.

**Reply by drjfloyd** [reply]
The User's Guide discusses how to define liquid droplets and how to define thermophysical properties of the liquid and gas phase for said droplets. It also provides you a list of species for which predefined gas and liquid properties exist. 

Any method for inserting particles (a sprinkler DEVC, INIT, PART_ID on MATL, PART_ID on SURF, etc.) can be used.
